collections-csharp/ConcurrentQueueInCSharp/ConcurrentQueueInCSharp.Console/Models/Order.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ConcurrentQueueInCSharp.Models
+{
+    public class Order
+    {
+        public string Id { get; set; }
```

Comment:
The `set` can be made `init`. It might be cleaner to even add a default value here, as the only time we are really setting this directly is in the tests, where we want to ensure we are getting back the same element we added. Even that case can be covered by just storing the ID value after creation as our expected result.

Reply:
Good point. I was originally thinking about using `init;` but ultimately settled down with this.